lib/main.dart

```diff
@@ -31,6 +33,7 @@ class _RandomWordsState extends State<RandomWords> {
   final _suggestions = <WordPair>[];
   final _saved = <WordPair>{};
   final _biggerFont = const TextStyle(fontSize: 18.0);
+  bool _rowColor = false;
```

Comment:
I would leave a comment what does this variable change, or either change the naming

Reply:
Fixed)

Reply:
`bool` variables are usually indicating something, so their name should ideally contain something like `is`, `has`, etc. Example: `array.isEmpty`. Your `_rowColor` name is more suitable to store colors, not flags

Reply:
Update)

Reply:
![photo_2021-09-27_21-12-49](https://user-images.githubusercontent.com/57919438/135315400-20768bc8-f7b2-487a-b61d-2d0e7eb6f337.jpg)
![photo_2021-09-27_21-12-44](https://user-images.githubusercontent.com/57919438/135315403-e3affcc1-e0cc-4126-867c-dac8638cdbfd.jpg)

Reply:
In the future, you should edit your PR description, instead of adding new messages to the conversation